packages/react-interactive-tab/src/components/InteractiveTab/useTabAnimatedIndicator.styles.ts

```diff
@@ -0,0 +1,173 @@
+import * as React from 'react';
```

Comment:
This file should be removed before merge, as we are going to import these hooks from the @fluentui/react-tabs once the https://github.com/microsoft/fluentui/pull/32125 merged and the package is published.